examples/write_link.rs

```diff
@@ -9,11 +9,11 @@ fn main() {
     println!("Generated keypair: {:?}", key);
     let privkey = PrivateKey::from_pkcs8(&key, SignatureScheme::Ed25519).unwrap();
 
-    let link = LinkMetadataBuilder::new()
-        .name(String::from("test"))
+    let link = LinkMetadataBuilder::new(None)
+        .builder_id(String::from("test"))
         .add_material(VirtualTargetPath::new("LICENSE".to_string()).unwrap())
         .add_product(VirtualTargetPath::new("Makefile".to_string()).unwrap())
-        .signed::<Json>(&privkey)
+        .signed::<Json>(MetaFormat::default(), &privkey)
```

Comment:
`signed` and `unsigned` need a `MetaFormat` to generate different version of attestation.